telosnetwork/telos-wallet
# Bugfix/token list reordering

# Fixes #139 

## Test scenarios
- log in with account `hyogaserelou` (or other account with proposal, dao, vote tokens)
- view wallet, table should not continuously shift order of tokens in table

## Checklist:
